css/css-font-loading/fontfaceset-has.html

```diff
@@ -18,7 +18,7 @@
 <script>
 test(function(t) {
     let fonts = document.fonts;
-    let font = fonts.keys().next().value;
+    let font = [...fonts.keys()][0];
```

Comment:
@tabatkins Was there a specific reason this change is needed? Unfortunately, it causes the test to fail in Gecko (with "fonts.keys() is not iterable"), although that's not the feature this test is meant to be addressing. Can we revert to Myles' previous syntax here?

The fact that `[...fonts.keys()]` doesn't work in Gecko would appear to be a separate interop issue that deserves its own test; it shouldn't be mixed up with whether `.has()` works.